Synthesis failed for Google.Cloud.Asset.V1

Hello! Autosynth couldn't regenerate Google.Cloud.Asset.V1. :broken_heart:

Here's the output from running `synth.py`:

```
autosynth [DEBUG] > Running: git log -1 --pretty=%at 3970c7a1fd8408f52ba892e10889f6124eba28a1
2020-11-16 22:36:42,578 autosynth [DEBUG] > Running: git log -1 --pretty=%at 81eef036ac0dae5c948555df2cc41f27f5730cc3
2020-11-16 22:36:42,581 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3faad13e606c56aacc9181944666a5205a509aaa
2020-11-16 22:36:42,584 autosynth [DEBUG] > Running: git log -1 --pretty=%at 30500d20b0b92e18050c2287b8682c94b313bc9f
2020-11-16 22:36:42,587 autosynth [DEBUG] > Running: git log -1 --pretty=%at a830e5d735942a23107840dae8c69eb18b1ea085
2020-11-16 22:36:42,590 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6c0f55f33f882cf97fd3135e93b144ad9c94ebab
2020-11-16 22:36:42,593 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8b159675150e6d8b149bc4db4540e574462bc4a2
2020-11-16 22:36:42,596 autosynth [DEBUG] > Running: git log -1 --pretty=%at 038351107de08b8f2796ab1fc3be5b84ea0c2d87
2020-11-16 22:36:42,600 autosynth [DEBUG] > Running: git log -1 --pretty=%at b1e1e0b13b580d8fb7a641978198fc8de7228b2f
2020-11-16 22:36:42,603 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b0ad15b0ff483486ae90d73092e7be00f8c1848
2020-11-16 22:36:42,606 autosynth [DEBUG] > Running: git log -1 --pretty=%at b7038c181dcaa75c4fe8270b842c941d1df3128c
2020-11-16 22:36:42,609 autosynth [DEBUG] > Running: git log -1 --pretty=%at 063e39ee72fd4134dd7fd14bad9966fef4e47e1b
2020-11-16 22:36:42,612 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7e17784e6465431981f36806e6376d69de1fc424
2020-11-16 22:36:42,615 autosynth [DEBUG] > Running: git log -1 --pretty=%at 20031e727d2b105b20c48492ab47fc4ff174a789
2020-11-16 22:36:42,618 autosynth [DEBUG] > Running: git log -1 --pretty=%at 96ca8bc1f621b7e972b958b35ad15af4c5476e16
2020-11-16 22:36:42,621 autosynth [DEBUG] > Running: git log -1 --pretty=%at 75d1324fff65abad6435e6d501e050439b5a616b
2020-11-16 22:36:42,624 autosynth [DEBUG] > Running: git log -1 --pretty=%at 0c9e3f8cb3a0c75983fe9a7897f0ef048d81e999
2020-11-16 22:36:42,627 autosynth [DEBUG] > Running: git log -1 --pretty=%at 836f0eaf5f21f300f63ac635e5ef263d183e0cdd
2020-11-16 22:36:42,630 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5d703efd4096375bb95c8492bac975f49f5da94d
2020-11-16 22:36:42,633 autosynth [DEBUG] > Running: git log -1 --pretty=%at eb60ffabbe50e6a4ba8f04c9a2fc0bfdb86433f7
2020-11-16 22:36:42,636 autosynth [DEBUG] > Running: git log -1 --pretty=%at be0bdf86cd31aa7c1a7b30a9a2e9f2fd53ee3d91
2020-11-16 22:36:42,640 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8d245ac97e058b541f1477b1a85d676b18e80849
2020-11-16 22:36:42,643 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1f8a5144b52f7677dc43c55b21ccaf9e1b425ceb
2020-11-16 22:36:42,646 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6a69c750c3f01a69017662395f90515bbf1fe1ff
2020-11-16 22:36:42,649 autosynth [DEBUG] > Running: git log -1 --pretty=%at 754a312a0d01cfc1484d397872ff45e5565af0da
2020-11-16 22:36:42,652 autosynth [DEBUG] > Running: git log -1 --pretty=%at c6b1c72942991b1c605fe56a6041e06a86a9443e
2020-11-16 22:36:42,656 autosynth [DEBUG] > Running: git checkout 342a9321ad81b8dd911f07a5b7d920fe6eb5510d
Note: checking out '342a9321ad81b8dd911f07a5b7d920fe6eb5510d'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 342a9321a feat: updated google.cloud.assuredworkloads.v1beta1.AssuredWorkloadsService service
2020-11-16 22:36:42,675 autosynth [DEBUG] > Running: git checkout c6b1c72942991b1c605fe56a6041e06a86a9443e
Note: checking out 'c6b1c72942991b1c605fe56a6041e06a86a9443e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at c6b1c729 fix: better fallback feature detection, jsdoc update
2020-11-16 22:36:42,689 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Asset.V1-56
2020-11-16 22:36:42,693 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1-56
Switched to branch 'autosynth-Google.Cloud.Asset.V1-56'
2020-11-16 22:36:42,712 autosynth [INFO] > Running synthtool
2020-11-16 22:36:42,712 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-11-16 22:36:42,713 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Asset.V1/56/sponge_log.log
2020-11-16 22:36:42,715 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-11-16 22:36:42,937 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py.
On branch autosynth-Google.Cloud.Asset.V1-56
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Asset.V1 for compatibility checking
Generating Google.Cloud.Asset.V1
Building new version of Google.Cloud.Asset.V1 for compatibility checking
Assets.cs(996,33): error CS0234: The type or namespace name 'OsConfig' does not exist in the namespace 'Google.Cloud' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/Google.Cloud.Asset.V1/Google.Cloud.Asset.V1.csproj]
Assets.cs(989,34): error CS0234: The type or namespace name 'OsConfig' does not exist in the namespace 'Google.Cloud' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/Google.Cloud.Asset.V1/Google.Cloud.Asset.V1.csproj]
2020-11-16 22:36:51,806 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Asset.V1:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Asset.V1')' returned non-zero exit status 1.
2020-11-16 22:36:51,850 autosynth [ERROR] > Synthesis failed
2020-11-16 22:36:51,850 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 342a9321a feat: updated google.cloud.assuredworkloads.v1beta1.AssuredWorkloadsService service
2020-11-16 22:36:51,881 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 354, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 189, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 334, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 65, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9264f6e7-1447-4846-8a90-439a32f4a9f8/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

--- a/apis/Google.Cloud.Asset.V1/Google.Cloud.Asset.V1/Google.Cloud.Asset.V1.csproj
+++ b/apis/Google.Cloud.Asset.V1/Google.Cloud.Asset.V1/Google.Cloud.Asset.V1.csproj
@@ -14,6 +14,7 @@
     <PackageReference Include="Google.Api.Gax.Grpc.GrpcCore" Version="[3.2.0, 4.0.0)" />
     <PackageReference Include="Google.Cloud.Iam.V1" Version="[2.0.0, 3.0.0)" />
     <PackageReference Include="Google.Cloud.OrgPolicy.V1" Version="[2.0.0, 3.0.0)" />
+    <PackageReference Include="Google.Cloud.OsConfig.V1" Version="[1.2.0, 2.0.0)" />
     <PackageReference Include="Google.Identity.AccessContextManager.Type" Version="[1.0.0, 2.0.0)" />
     <PackageReference Include="Google.Identity.AccessContextManager.V1" Version="[1.0.0, 2.0.0)" />
     <PackageReference Include="Google.LongRunning" Version="[2.0.0, 3.0.0)" />

--- a/apis/apis.json
+++ b/apis/apis.json
@@ -74,6 +74,7 @@
         "Google.LongRunning": "2.0.0",
         "Google.Cloud.Iam.V1": "2.0.0",
         "Google.Cloud.OrgPolicy.V1": "2.0.0",
+        "Google.Cloud.OsConfig.V1": "1.2.0",
         "Grpc.Core": "2.31.0"
       },
       "tags": [

